__init__.py

```diff
@@ -1,7 +1,7 @@
 bl_info = {
     "name": "AmbientCG Material Importer",
     "author": "Nino Filiu",
-    "version": (1, 1, 0),
+    "version": (1, 1, 2),

```

Comment:
Feature improvement so

```suggestion
    "version": (1, 2, 0),

```